README.md

```diff
@@ -15,6 +15,7 @@ curl https://krew.sh/preflight | bash
  and run:
  
 ```
+# It is ONLY an example. Do **not** use to validate real scenarios.
```

Comment:
I'd like to move this out of the command example and have it directly above.  e.g.:

"and run:" becomes

"and run, where  https://preflight.replicated.com provides an example preflight spec:"

we should also add a note to "To run a sample preflight check from a sample application,":

"To run a sample preflight check from a sample application (NOTE: this is an example. Do **not** use to validate real scenarios.):"

Reply:
changed let me wdyt?